src/components/Button/button.css

```diff
@@ -25,7 +25,7 @@
     background-color: rgb(0, 139, 109);
 }
 
-@media only screen and (max-width: 64em) {
+@media only screen and (max-width: 64em), (max-height: 46.3125em)  {
```

Comment:
This kind of media query seems really brittle, but we can improve on this later.

Reply:
I agree with you in the point of the max-height media query is not the best one, I do it because the widget is large and in some screens, like for example 1550x722 the widget doesn't fit on the screen.

Regarding the choice of `em` for media-queries which alternative do you think is best? I think that is the best and the most consistent unit to use in media queries

Reply:
Ah I was misinformed about the em unit in the context of media queries, I think it's fine here (but probably not that necessary on modern browsers)! But yeah the more we can get the styles can naturally scale to the browser dimensions without needing to corner case various screen sizes, the better IMO.